wagtail/admin/models.py

```diff
@@ -1,5 +1,55 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Count
+
+from modelcluster.fields import ParentalKey
+from taggit.models import Tag
+
 # The edit_handlers module extends Page with some additional attributes required by
 # wagtailadmin (namely, base_form_class and get_edit_handler). Importing this within
 # wagtailadmin.models ensures that this happens in advance of running wagtailadmin's
 # system checks.
 from wagtail.admin import edit_handlers  # NOQA
+from wagtail.core.models import Page
```

Comment:
Your comment on https://github.com/wagtail/wagtail/issues/5507#issue-480767983 mentions that

> 'models' is a special case within Django, as it gets preloaded on startup - will this cause any side effects?

I am not that familiar with this behavior, so don't know what side effects might happen here, but I do think that using models.py as the filename for this might cause some confusion. Developers might expect admin/models.py to contain Django models for the admin app, as opposed to utility methods about models.

Could we name this file something else, maybe model_usage.py or usage.py?

Reply:
Having pondered this some more... if wagtail.admin.models _did_ have any models of its own, I wouldn't have any second thoughts about putting these functions alongside them (just as, say, wagtail.documents.models has `get_document_model`) - and I don't think the situation is substantially different when models.py is otherwise empty. So I think this is fine - it just means that the functions get loaded slightly earlier in the startup sequence.